src/PixiEditor/Data/Localization/Languages/fr.json

```diff
@@ -1,1135 +1,691 @@
+  "FINISHED": "Fini",
+  "GENERATING_SPRITE_SHEET": "Génération de la Feuille de Sprite",
+  "RENDERING_IMAGE": "Image de Rendu",
+  "PROGRESS_POPUP_TITLE": "Progrès",
+  "POINTS": "Points",
+  "MIN_DISTANCE": "Distance Min.",
+  "MAX_POINTS": "Points Max.",
+  "DISTRIBUTE_POINTS": "Distribution des points",
+  "REMOVE_CLOSE_POINTS": "Enlever les points proches",
+  "RASTERIZE_SHAPE": "Rastériser la forme",
+  "MODE": "Mode",
+  "Factor": "Facteur",
+  "NORMALIZE": "Normaliser",
+  "WEIGHT_FACTOR": "Poids",
+  "STARS_EXAMPLE": "Étoiles",
+  "ADD_EMPTY_FRAME": "Ajouter une frame vide",
+  "DUPLICATE_FRAME": "Dupliquer la frame",
+  "DELETE_FRAME": "Enlever la frame",
+  "DEFAULT_MEMBER_NAME": "Nouvel élément",
+  "SELECT_FILE_FORMAT": "Sélectionner le format de fichier",
+  "NEW_PALETTE_FILE": "palette",
+  "ISLAND_EXAMPLE": "Îles",
+  "TOGGLE_ANIMATION": "Activer/Désactiver l'animation",
+  "NEW_FROM_CLIPBOARD": "Nouveau depuis le presse-papier",
+  "OFFSET": "Décalage",
+  "SHAPE": "Forme",
+  "STRUCTURE": "Structure",
+  "NUMBERS": "Nombres",
+  "OPERATIONS": "Opérations",
+  "GENERATION": "Génération",
+  "NUMBER": "Nombre",
+  "ANIMATION": "Animation",
+  "POSITION": "Position",
+  "MATH_ADD": "Ajouter",
+  "MATH_SUBTRACT": "Soustraire",
+  "MULTIPLY": "Multiplier",
+  "DIVIDE": "Diviser",
+  "SIN": "Sin",
+  "COS": "Cos",
+  "TAN": "Tan",
+  "PIXEL_ART_TOOLSET": "Pixel Art",
+  "VECTOR_TOOLSET": "Vecteur",
+  "VECTOR_LAYER": "Couche de Vecteur",
+  "SYNC_WITH_PRIMARY_COLOR_LABEL": "Synchroniser avec la couleur principale",
+  "NEW_ELLIPSE_LAYER_NAME": "Ellipse",
+  "NEW_RECTANGLE_LAYER_NAME": "Rectangle",
+  "NEW_LINE_LAYER_NAME": "Ligne",
+  "PAINT_TOOLSET": "Peinture",
+  "TOGGLE_SNAPPING": "Activer le calage",
+  "VIEWPORT_ROTATION": "Rotation de l'affichage",
+  "CANVAS_POSITION": "Position du canevas",
+  "CANVAS": "Canevas",
+  "UNEXPECTED_SHUTDOWN_MSG": "PixiEditor a cessé de fonctionner de manière inattendue. Nous avons chargé la dernière sauvegarde automatique de vos fichiers.",
+  "OPEN_AUTOSAVES": "Parcourir les sauvegardes automatiques",
+  "AUTOSAVE_SETTINGS_HEADER": "Sauvegarde automatique",
+  "AUTOSAVE_SETTINGS_SAVE_STATE": "Réouvrir les derniers fichiers au démarrage",
+  "AUTOSAVE_SETTINGS_PERIOD": "Fréquence de sauvegarde automatique",
+  "AUTOSAVE_ENABLED": "Sauvegarde automatique activée",
+  "MINUTE_UNIVERSAL": "min",
+  "AUTOSAVE_SETTINGS_SAVE_USER_FILE": "Sauvegarder automatiquement le fichier sélectionné",
+  "PIXEL_PERFECT_OUTLINE_TYPE": "Pixel perfect",
+  "AUTOSAVE_OPEN_FOLDER": "Ouvrir le dossier des sauvegardes automatiques",
+  "AUTOSAVE_OPEN_FOLDER_DESCRIPTIVE": "Ouvre le dossier de stockage des sauvegardes automatiques",
+  "AUTOSAVE_TOGGLE_DESCRIPTIVE": "Active/désactive la sauvegarde automatique",
+  "TEXT_TOOL_ACTION_DISPLAY": "Cliquez sur le canvas pour ajouter un nouveau texte (glissez en cliquant pour définir la taille). Cliquez sur un texte existant pour le modifier."
```

Comment:
Something is not right here @CPKreu, why did it add less lines than on master? I just added this file to master manually from poeditor